recipes/catch2/2.x.x/conanfile.py

```diff
@@ -1,9 +1,11 @@
-from conans import ConanFile, CMake, tools
-from conans.errors import ConanInvalidConfiguration
-import functools
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.cmake import CMake, CMakeToolchain, cmake_layout
+from conan.tools.files import copy, get, rmdir, replace_in_file
+from conan.tools.scm import Version
 import os
 
-required_conan_version = ">=1.43.0"
+required_conan_version = ">=1.52.0"
```

Comment:
```suggestion
required_conan_version = ">=1.53.0"
```
https://github.com/conan-io/conan/issues/12085